Client/InfluxDBClient.cs

```diff
@@ -312,15 +313,16 @@ public class InfluxDBClient : AbstractRestClient, IInfluxDBClient
         /// Create a instance of the InfluxDB 2.x client.
         /// </summary>
         /// <param name="options">the connection configuration</param>
-        public InfluxDBClient(InfluxDBClientOptions options)
+        /// <param name="client">the http client</param> 
+        public InfluxDBClient(InfluxDBClientOptions options, HttpClient client = null)
```

Comment:
Please add `HttpClient` as a part of `InfluxDBClientOptions`. It will be more consistent.

Reply:
Yes, it will be more appropriate as part of the InfluxDBClientOptions.

Reply:
Thanks